src/components/autonomous/WaypointNav.js

```diff
@@ -18,6 +20,17 @@ function WaypointNav() {
     dispatch(requestWaypointNav(formJson));
   };
 
+  function grabFromClipboard () {
+    navigator.clipboard.readText().then(text => {
+      if(text.match("-?\\d{1,}\\.?\\d{0,}, -?\\d{1,}\\.?\\d{0,}")) {
+        setLat(text.split(", ")[0]);
+        setLon(text.split(", ")[1]);
+      }
+    }).catch(err => {
+      console.error("Failed to read clipboard contents: ", err);
```

Comment:
This should maybe alert in addition to printing, if the user accidentally clicks "deny" on the permissions, they should be alerted that this functionality is failing.

Reply:
How should I alert the user when this happens? Is there some function that I should use or should I create another element below the button that displays the status or something?

Reply:
We could just use the [`alert()`](https://www.w3schools.com/jsref/met_win_alert.asp) function